### PR TITLE
Report CLI tests to teamcity

### PR DESCRIPTION
**Pull request by Hypnosphi**
Issue: master is red

## What I did

Added teamcity service mesages to CLI tests which allows teamcity to recognise them as separate [testcases](https://teamcity.jetbrains.com/viewLog.html?buildId=1725425&buildTypeId=OpenSourceProjects_Storybook_CliTest&tab=testsInfo). Then I muted the `sfc_vue` test to make build green and assigned investigation of the failure to @igor-dv

### CODE REVIEW

File: lib/cli/test/run_tests.sh [hunk 1]
```diff
@@ -49,7 +54,27 @@ for dir in *
...
+  if [ $teamcity -eq 1 ]
+  then
+    if [ $failed -eq 1 ]
+    then
+      echo "##teamcity[testFailed name='$dir']"
+    fi
+    echo "##teamcity[testFinished name='$dir']"
+  elif [ $failed -eq 1 ]
+  then
+    exit 0
```
Comment (shilman) [inline lib/cli/test/run_tests.sh]: Don't we want a failing exit code in this case?

**Review by shilman — CHANGES_REQUESTED** [review]

File: lib/cli/test/run_tests.sh (hunk 1 above)
Comment (Hypnosphi) [reply lib/cli/test/run_tests.sh]: sure, my mistake

Comment (Hypnosphi) [reply lib/cli/test/run_tests.sh]: Fixed

**Review by shilman — APPROVED** [review]
LGTM!

**Comment by igor-dv** [comment]
Looks like tests are failing now in master (the `Tests`) https://teamcity.jetbrains.com/viewType.html?buildTypeId=OpenSourceProjects_Storybook_Test&branch_OpenSourceProjects_Storybook=%3Cdefault%3E&tab=buildTypeStatusDiv

**Comment by Hypnosphi** [comment]
Weird, `yarn test --core --teamcity` works OK locally

**Comment by Hypnosphi** [comment]
Seems like node 11 is the reason. I'll enforce node 10

**Comment by Hypnosphi** [comment]
https://github.com/storybooks/storybook/pull/4720

**Pull request merged**